src/main/java/use_cases/create_curriculum/CreateCurriculumPresenter.java

```diff
@@ -9,13 +11,21 @@
  */
 public class CreateCurriculumPresenter implements CreateCurriculumOutputBoundary {
 
+    private final List<DashboardUiInterface> dashboardObservers;
+
+    public CreateCurriculumPresenter(List<DashboardUiInterface> dashboardObservers) {
+        this.dashboardObservers = dashboardObservers;
+    }
+
     /**
      * Prepares the created curriculum for presentation
      *
      * @param curriculumModel the newly created Curriculum's model
      */
     @Override
     public void displayCurriculum(CurriculumModel curriculumModel) {
-
+        String successfullyCreatedCurriculumMsg = String.format("The %s curriculum (i.d. %d) was successfully created",
+                curriculumModel.getCurriculumName(), curriculumModel.getCurriculumID());
+        this.dashboardObservers.get(0).createdCurriculum(successfullyCreatedCurriculumMsg);
     }
```

Comment:
Good, clear implementation here. The string simply puts out the curriculum Name and Curriculum ID together with the message.
Also, I see you have implemented the Observers view which will be a massive help after explaining it to us today.

Reply:
This message will also be displayed on the UI end so that the user knows the curriculum was been successfully created